README.md

```diff
@@ -1,18 +1,27 @@
 # cortx-test
-CORTX-TEST is an automation repository for multiple automation projects developed for LDR R2 and future versions. 
-Right now, it can be divided into following logical loosely coupled parts 
-* Test framework. 
-* Test execution framework. 
-* Robot framework and 
+CORTX-TEST is a repository for multiple automation projects developed for CORTX (QSG Ref: https://github.com/Seagate/cortx/blob/main/README.md#get-started) and supported solutions/systems.
+
+It is logically divided into following components: 
+* Test Automation framework
+* Execution Ecosystem
```

Comment:
If we can keep same words like Test Execution Framework.  Execution Ecosystem is more broader and it includes targets and Labs.

Reply:
OK, let me change it back to TE Framework.